MovieApp/MovieApp/View/Main View/MovieCell.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+
+class MovieCell : UITableViewCell {
+
+    var mainView : UIView!
+    var movieImageView : UIImageView!
+    var titleLabel : UILabel!
+    var descriptionLabel : UILabel!
+    static let cellIdentifier = "MovieCell"
```

Comment:
You can write Identifiers like this
```swift
static let reuseIdentifier = String(describing: MovieCell.self)
```
static let variables are declared on top of the class by style guide